total rehaul with HUGO

omega7379 opened the pull request:
Added all the content required, instructions, links...etc. Bonus fluff coming at a later date possibly.

omega7379 committed:
update quickstart

omega7379 committed:
content additions

adding more build instructions, types of tracking available. Added FAQ page. Added links to contributors

omega7379 committed:
source update

added special instructions for building from source.

omega7379 committed:
multiple file changes

added FAQ page to navigation bar, demo added to about page, built FAQ content,  hardware requirements, added tracking information, ToC to source page

omega7379 committed:
README.md

adding readme file

omega7379 committed:
fix buttons

omega7379 committed:
Merge branch 'refactor/hugo-rewrite' of https://github.com/omega7379/virtual-puppet-project-website into refactor/hugo-rewrite

omega7379 committed:
Merge branch 'refactor/hugo-rewrite' of https://github.com/omega7379/virtual-puppet-project-website into refactor/hugo-rewrite

omega7379 committed:
quick edits

rename hyperlink, add prerequisites

you-win commented on content/quickstart.md:
```diff
@@ -39,8 +83,74 @@ Under construction.
+
+4- use the tracker in the app!
+
+#### Manual Build
```
I'm against putting developer documentation on the website if it's already available in the repository's README. Technical documentation becomes out-of-date very quickly

you-win commented on content/quickstart.md:
```diff
@@ -7,29 +8,72 @@ tocopen: true
 # Prerequisites
 
-Under construction.
+1- OpenSeeFace tracking: webcam
```
In markdown, you can get an ordered/unordered list by doing either

"1."

or

"-"

You should avoid brute forcing lists like this. Note that sentences in lists should still start with a capital letter but periods can be optional as long as you consistently use/omit them

you-win commented on content/quickstart.md:
```diff
@@ -7,29 +8,72 @@ tocopen: true
+
+Sorry, no build provided :( feel free to build from source!
+
+click here for [instructions!]({{< ref "source/#building-from-source">}})
```
This should be a link to the repo's README

you-win commented on content/quickstart.md:
```diff
@@ -39,8 +83,74 @@ Under construction.
+
+Not included by default. Follow instructions to add it!
+
+#### Prebuilt release
```
Make this the default and only instruction, technical instructions should be located on GitHub

you-win commented on content/source.md:
```diff
@@ -2,6 +2,8 @@
 title: Source
 url: source/
 summary: Source for the Virtual Puppet Project
+showToc: true
```
Not necessary since these details should only be found on GitHub

you-win commented on content/quickstart.md:
```diff
@@ -7,29 +8,72 @@ tocopen: true
+
+installation for [arch](https://aur.archlinux.org/packages/puppeteer)
+
+~~Other Types Not Provided~~, package managers are made by the community :)
```
Double tilde means that the text in-between is crossed out, which doesn't make any sense in this context.

> package managers are made

Should be "distribution-specific packages are maintained". A package manager is something like `apt`, `yum`, `pacman`, etc

you-win commented on content/quickstart.md:
```diff
@@ -7,29 +8,72 @@ tocopen: true
-Under construction.
+#### OpenSeeFace-GD (old)
+
+installation for [arch](https://aur.archlinux.org/packages/puppeteer)
```
Sentence should start with a capital letter, "arch" should be "Arch Linux" since arch is a generic computer term that stands for "architecture" (e.g. x86, aarch)

you-win commented on content/about.md:
```diff
@@ -5,3 +5,19 @@ summary: About the Virtual Puppet Project
+
+## Demo
+
+enjoy a quick demo from a slightly older alpha version by [@ItsRogueRen](https://youtu.be/7K63acAQj7Q)
```
Sentence should start with a capital letter and end with a period.

you-win commented on content/FAQ.md:
```diff
@@ -0,0 +1,47 @@
+Simply put, it's a very niche thing and not a lot of people will be using it. If enough people ask about it, it might be
+considered later.
+
+## What is engine is VPupPr using
```
There is an extra "is" here

you-win commented on content/FAQ.md:
```diff
@@ -0,0 +1,47 @@
+tocopen: true
+---
+
+## I read the docs but still need help!
```
Headings should all either be Title Case or Sentence case

you-win commented on README.md:
```diff
@@ -0,0 +1,4 @@
+# virtual-puppet-project-website
+
+The source code for the Virtual Puppet Project (VPupPr) website.</br>
```
Avoid using a raw \<br> tag here. Markdown supports newlines if you add an extra newline between sentences.

you-win commented on content/quickstart.md:
```diff
@@ -7,29 +8,72 @@ tocopen: true
-# Installation
+2- IFacialMocap tracking: iOS device with face ID or A12 (or newer) chip
+
+3- MeowFace tracking: android device that supports ARCore (most modern phones do)
```
"Android" is a proper noun

you-win commented on content/quickstart.md:
```diff
@@ -7,29 +8,72 @@ tocopen: true
+1- OpenSeeFace tracking: webcam
 
-# Installation
+2- IFacialMocap tracking: iOS device with face ID or A12 (or newer) chip
```
> face ID or A12 (or newer) chip

This is redundant. Would be better to specify a specific model of Apple device, in this case the iPhone X

you-win commented on content/quickstart.md:
```diff
@@ -7,29 +8,72 @@ tocopen: true
+3- MeowFace tracking: android device that supports ARCore (most modern phones do)
 
-Under construction. Grab something from [downloads]({{< ref "downloads/">}}) :)
+4- 64-bit Operating system with x86 architecture
```
Technically wrong, it should be x86_64 (intel) or amd64 (amd). x86 is 32 bit. Would be better to leave out the architecture entirely or leave it vague at "Any mainstream 64-bit operating system"

you-win commented on content/quickstart.md:
```diff
@@ -7,29 +8,72 @@ tocopen: true
 ## Windows
 
-Under construction. Grab something from [downloads]({{< ref "downloads/">}}) :)
+1- download windows [zip file](https://github.com/virtual-puppet-project/vpuppr/releases)
```
Sentences should be capitalized. However in a list, sentences do not necessarily need to end in a period

you-win commented on content/quickstart.md:
```diff
@@ -7,29 +8,72 @@ tocopen: true
+
+2- unzip files into a directory, (do not move files!)
+
+3- run the ``.exe`` on windows
```
Double "` " is not necessary, a single tilde is enough

you-win commented on content/quickstart.md:
```diff
@@ -39,8 +83,74 @@ Under construction.
+
+### MeowFace
+
+An android version of IFacialMocap. Also supports perfect sync models.
```
Android is a proper noun.

"IFacialMocap" should be "iFacialMocap" as that is the official title on their website

you-win commented on content/quickstart.md:
```diff
@@ -39,8 +83,74 @@ Under construction.
+
+### IFacialMocap
+
+Uses the 3D tracking capabilities of Apple Inc devices to provide optimal tracking data from your phone. Use if your model has
```
"Inc" should have a "." to indicate that it is an abbreviation

> Use if your model has perfect sync.

Should be "Use this if ..."

you-win commented on content/quickstart.md:
```diff
@@ -39,8 +83,74 @@ Under construction.
+### IFacialMocap
+
+Uses the 3D tracking capabilities of Apple Inc devices to provide optimal tracking data from your phone. Use if your model has
+perfect sync.
```
Perfect Sync is a proper noun

you-win commented on content/quickstart.md:
```diff
@@ -39,8 +83,74 @@ Under construction.
+
+An android version of IFacialMocap. Also supports perfect sync models.
+
+MeowFace is a precision face tracking application that can send 42 ARKit-like blendshapes as well as head rotation tracking to PC VTuber applications. -MF developers
```
Quote should be encased in double quotes. Should be removed since this could go out-of -date very quickly

"MF" should be MeowFace since the acronym has not been defined any else on the page

you-win commented on content/quickstart.md:
```diff
@@ -39,8 +83,74 @@ Under construction.
+Uses the 3D tracking capabilities of Apple Inc devices to provide optimal tracking data from your phone. Use if your model has
+perfect sync.
+
+iFacialMocap is an app that captures facial movements with an iPhone equipped with FaceID. This app sends the captured motion in real time to 3DCG software such as Maya, Unity, Blender on PC. -IFM developers
```
Quote should be encased in double quotes. Would be better to leave it out since this could go out-of-date very quickly

"IFM" should be expanded to iFacialMocap since the acronym has not been defined anywhere else on the page

you-win commented on content/quickstart.md:
````diff
@@ -7,29 +8,72 @@ tocopen: true
-Under construction. Grab something from [downloads]({{< ref "downloads/">}}) :)
+Download the linux [zip file](https://github.com/virtual-puppet-project/vpuppr/releases)
+
+```
````
Should have a sub heading or at least a description saying that this is example output from a terminal

you-win commented on content/quickstart.md:
```diff
@@ -7,29 +8,72 @@ tocopen: true
+    $ chmod +x vpuppr.x86_64
+    $ cd resources/extensions/open_see_face/OpenSeeFaceFolder/OpenSeeFace/
+    $ chmod +x facetracker
+    To run-
```
I'm not a fan of raw text mixed in with terminal output. Should be split into another code block

you-win commented on content/FAQ.md:
```diff
@@ -0,0 +1,47 @@
+
+## What is engine is VPupPr using
+
+Currently VPupPr runs on godot 3.x with plans to port over to 4.x once the beta becomes more stable.
```
"godot" is a proper noun

you-win commented on content/FAQ.md:
```diff
@@ -0,0 +1,47 @@
+
+## What is VPupPr
+
+VPupPr is an open-source project that animates 3D objects using the glb or vrm file extension. 
```
> animates 3D objects using the glb or vrm file extension.

Could be reworded as: animates `glb` and `vrm` models. Backticks should be used to denote file formats.

you-win commented on content/FAQ.md:
```diff
@@ -0,0 +1,47 @@
+## What is VPupPr
+
+VPupPr is an open-source project that animates 3D objects using the glb or vrm file extension. 
+Most notable for using 3D vtube models as its main usage.
```
Sentence feels awkward, could be reworded as: It is most notable for its use as a tracker and renderer for VTuber models.

"Most notably" could be used, but the adjective is supposed to allude to something mentioned in the previous sentence. In this case, there is nothing that it is contrasting with or expanding on that relates to notability.

you-win commented on content/FAQ.md:
```diff
@@ -0,0 +1,47 @@
+
+## Platform Supported
+
+Currently supported on windows and common linux distributions. MacOS/OSX is not currently supported,
```
"windows" and "linux" are proper nouns. Just OSX should be used when referring to the Apple desktop OS.

"however" should be inserted after the comma to make it a compound sentence

you-win commented on content/FAQ.md:
```diff
@@ -0,0 +1,47 @@
+## Platform Supported
+
+Currently supported on windows and common linux distributions. MacOS/OSX is not currently supported,
+feel free to fork the github [repo](https://github.com/virtual-puppet-project/vpuppr) and offer it to
```
"offer it to the community" has a weird vibe to it. I would personally leave it out since most of this section refrains from commentary.

you-win commented on content/FAQ.md:
```diff
@@ -0,0 +1,47 @@
+
+## Building from Source
+
+VPupPr runs off a custom godot engine. Get the pre-compiled engine [here](https://github.com/you-win/godot/releases).
```
> VPupPr runs off a custom godot engine

Should be "off of a custom build of the Godot Engine"

> Get the pre-compiled engine

Should be "Get a precompiled build of the engine ..."

you-win commented on content/FAQ.md:
```diff
@@ -0,0 +1,47 @@
+## Building from Source
+
+VPupPr runs off a custom godot engine. Get the pre-compiled engine [here](https://github.com/you-win/godot/releases).
+You can find the instructions [here]({{< ref "source/#building-from-source">}})
```
End sentences with a period

you-win commented on content/FAQ.md:
```diff
@@ -0,0 +1,47 @@
+
+Please note it might take a second for the tracking to activate. It _is_ trying to map your face! Also check that
+your face is well lit, usually having a lamp above your desk is enough. If it's still not working, submit an issue
+and/or ask the discord community!
```
"discord" is a proper noun

you-win commented on content/about.md:
```diff
@@ -5,3 +5,19 @@ summary: About the Virtual Puppet Project
+
+Having issues with the program? Submit an issue [here](https://github.com/virtual-puppet-project/vpuppr/issues)!
+
+Feel free to join the [discord](https://discord.gg/PeHR6Tjyvn) community to share your thoughts or help collaborate!
```
"discord" is a proper noun

omega7379 committed:
pr changes

requested changes have been fixed/adjusted for the PR

omega7379 committed:
spell check

checking for errors

omega7379 committed:
direct downloads and README update

added direct download for windows and linux zip files. Added instructions to run website
